app/controllers/AccountManagement.scala

```diff
@@ -353,12 +354,13 @@ object AccountManagement extends Controller with ContextLogging with CatalogProv
 
     val futureMaybeFutureManagePage = for {
       subscription <- OptionT(eventualMaybeSubscription).filter(!_.isCancelled)
+      account <- OptionT(tpBackend.zuoraService.getAccount(subscription.accountId).map{Some(_)}.recover{case t: Throwable => None})
       pendingHolidays <- OptionT(tpBackend.suspensionService.getUnfinishedHolidays(subscription.name, now).map(_.toOption))
-      billingSchedule <- OptionT(tpBackend.commonPaymentService.billingSchedule(subscription.id, subscription.accountId, numberOfBills = 13).map(Some(_):Option[Option[BillingSchedule]]))
+      billingSchedule <- OptionT(tpBackend.commonPaymentService.billingSchedule(subscription.id, account, numberOfBills = 13).map(Some(_):Option[Option[BillingSchedule]]))
```

Comment:
So we aren't actually making an extra call here, we just use [this version](https://github.com/guardian/membership-common/blob/e9e2118695c7c5d52463b8a7300a339795bdf959/src/main/scala/com/gu/memsub/services/PaymentService.scala#L93) of billingSchedule instead of the old one.